### DISCUSSION TITLE
Using Airodump-ng

### DISCUSSION THREAD

**Discussion by Carlos-Izagui** [aircrack-ng/aircrack-ng#2583] [Q&A]
I have monitor enabled on my wlan0 but after I run the command airodump-ng wlan0 I don't pick up any BSSID's even though I can see channels being scanned. Can anyone tell me what can be done SOLVE THIS?

**Comment by Mister-X-** [comment]
Try this:

1. Telling us what driver and chipset you are using, as well as the Linux distro and kernel version
2. Ensure you have killed network managers prior to putting your adapter in monitor mode
3. check dmesg for any error related to that driver

**Comment by rodface365** [comment]
GPSd won't work with airodump-ng, is there a fix available?
